components/suggest/src/schema.rs

```diff
@@ -643,6 +637,10 @@ impl ConnectionInitializer for SuggestConnectionInitializer<'_> {
                 )?;
                 Ok(())
             }
+            36 => {
```

Comment:
This should be `35`, it's the version we're upgrading from, not to. That should fix one of the test failures.